packages/service-library/tests/test_rest_pagination_utils.py

```diff
@@ -0,0 +1,111 @@
+from copy import deepcopy
+
+import pytest
+from servicelib.rest_pagination_utils import (
+    PageLinks,
+    PageMetaInfoLimitOffset,
+    PageResponseLimitOffset,
+)
+from yarl import URL
+
+
+def test_page_response_limit_offset_model():
```

Comment:
yes. just thought why parametrize when there is nothing else?

Reply:
Yes, you have a point.  I use it by default so that next time I have a new model I simplly add it to the list ...
Sorry, I took the wrong example. FYI: check the fixture ``model_cls_examples`` extracts all examples inside. This allows then to deal with all lexamples of a set of models
```python
@pytest.mark.parametrize("model_cls", (Meta,))
def test_meta_model_examples(model_cls, model_cls_examples):
    for name, example in model_cls_examples.items():
        print(name, ":", pformat(example))
        model_instance = model_cls(**example)
        assert model_instance, f"Failed with {name}"
```

Reply:
ok done... found a case;)